core/app/models/spree/address.rb

```diff
@@ -9,7 +9,7 @@ class Address < Spree::Base
     has_many :cartons, inverse_of: :address
     has_many :credit_cards, inverse_of: :address
 
-    validates :firstname, :lastname, :address1, :city, :country, presence: true
+    validates :firstname, :lastname, :address1, :city, :country_id, presence: true
```

Comment:
Good catch. That was a very tactical fix for some broken test that I've forgotten. I also ran across the `validate: true` option to `belongs_to` today which should also handle the situation. Is there a preference?